index.html

```diff
@@ -19,10 +19,11 @@
             <a class="logo">Femi</a>
             <a class="togglebar" href="#"><img src="Images/Icon-Menu.png" alt="menu icon"></a>
             <ul class="nav-items">
-                <li><a href="#">portfolio</a></li>
-                <li><a href="#">about</a></li>
-                <li><a href="#">contact</a></li>
-                <li><a href="#"><img src="Images/Icon-Mail.png" alt="mail icon"></a></li>
+                <li class="close"><a href="#"><img src="Images/Icon-Cancel.png" alt="Cancel Icon"></a></li>
+                <li class="show-list"><a href="#portfolio">Portfolio</a></li>
+                <li class="show-list"><a href="#about">About</a></li>
+                <li class="show-list"><a href="#contact">Contact</a></li>
```

Comment:
- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.